src/compiler/diagnosticMessages.json

```
@@ -2607,6 +2607,10 @@
         "category": "Error",
         "code": 2689
     },
+    "'{0}' only refers to a type, but is being used as a value here. Did you mean to use 'K in {0}'?": {

```

What if the type on the RHS is named `K`?

---

which message would be better for this case?

---

```suggestion
    "'{0}' only refers to a type, but is being used as a value here. Did you mean to use '{1} in {0}'?": {
```

Pick `K` for slot 1 unless slot 0 is `K`; in that case, use `P`.